File: stress/t/opt_float_licm.f
      PROGRAM OFLICM
      IMPLICIT DOUBLE PRECISION (A-H,O-Z)
      INTEGER*8 I
      A=3.0D0
      B=1.5D0
      S=0.0D0
      I=0
   10 IF (I.GE.100) GOTO 20
      T=A*B+SQRT(A)
      S=S+T
      I=I+1
      GOTO 10
   20 CONTINUE
      WRITE(*,'(A,F0.6)') 's=',S
      WRITE(*,'(A,F0.6)') 't=',T
      END

File: stress/t/opt_float_licm.c
#include <stdio.h>
#include <math.h>
int main(void){
  double a=3.0,b=1.5,s=0.0,t=0.0;
  long i;
  for(i=0;i<100;i=i+1){
    t=a*b+sqrt(a);
    s=s+t;
  }
  printf("s=%f\n",s);
  printf("t=%f\n",t);
  return 0;
}
